package-scripts.js

```diff
@@ -42,22 +44,8 @@ module.exports = {
     },
     validate: {
       description: 'This runs several scripts to make sure things look good before committing or on clean install',
-      script: concurrent({
-        'build-and-validate': {
-          script: 'nps build.andValidate',
-          color: 'bgBlue.bold',
-        },
-        test: {
-          script: 'nps test',
-          color: 'bgMagenta.bold',
-        },
-        'lint-staged': {
```

Comment:
`lint-staged` shouldn't happen during the `validate` script. Only during `precommit`, so I've moved it to that.